skabase/SKABaseDevice/SKABaseDevice.py

```diff
@@ -50,15 +48,18 @@ class SKABaseDevice(with_metaclass(DeviceMeta, Device)):
     # PROTECTED REGION ID(SKABaseDevice.class_variable) ENABLED START #
 
     def _init_syslog(self):
+        self.formatter = logging.Formatter('%(name)s: %(levelname)s %(module)s %(message)r')
         try:
             self.logger = logging.getLogger(__name__)
-            #self.syslogs = SysLogHandler(address='/dev/log', facility='syslog')
+            # self.syslogs = SysLogHandler(address='/dev/log', facility='syslog')
             self.syslogs = SysLogHandler(address='/var/run/rsyslog/dev/log', facility='syslog')
```

Comment:
If your address is /var/run/rsyslog/dev/log, why do you make the link in the docker compose file? I am referring to the command "sudo ln -sf /var/run/rsyslog/dev/log /dev/log"

Reply:
There was some issue faced by other team members so path was changed from /dev/log to this one.